### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by FinalAngel**
* Added test framework
* Adapted root files such as ``README``, ``CHANGELOG``, ``setup.py`` and others to conform with other core addons such as django CMS Picture

### CODE REVIEW

File: addon.json [hunk 1]
```diff
@@ -2,9 +2,5 @@
     "package-name": "aldryn-bootstrap3",
     "installed-apps": [
         "aldryn_bootstrap3"
-    ],
-    "protected": [
-        "templates/*",
-        "static/*"
     ]
```
Comment (Chive) [inline addon.json]: are you sure about this? I think this still required?

Comment (Chive) [inline addon.json]: ^^ Please check this before merging

Comment (vxsx) [inline addon.json]: don't think we should remove this

Comment (FinalAngel) [inline addon.json]: @vxsx @Chive I reverted the changes as @stefanfoulis wasn't sure either

**Pull request merged**